### PR TITLE
remove references to Cocoapods 0.39 (fixes issue #214)

### PR DESCRIPTION
**Pull request by gurgeous**

### CODE REVIEW

**Comment by gurgeous** [comment]
I signed it!

**Comment by khandpur** [comment]
Hello, thanks for sending this PR. Could you elaborate on the reason behind it? Is this in reference to https://github.com/google/EarlGrey/issues/214 ?

**Comment by bootstraponline** [comment]
Did you try:

> If you don't want a gem dependency, replace require 'earlgrey' with load 'configure_earlgrey.rb' and manually copy configure_earlgrey.rb from the gem into the same directory as the Podfile.

**Comment by tirodkar** [comment]
@gurgeous, thanks for this, but we don't want to remove the old Podfile section just yet. Remember, you can still use EarlGrey with your 0.38 or so, just without the gem.

I'd recommend adding a change similar to what @bootstraponline stated in Point 2 here -

https://github.com/google/EarlGrey/blob/master/docs/install-and-run.md#step-2-add-earlgrey-as-a-framework-dependency

You could change it to - 

> Because EarlGrey must be embedded within the app under test, we need to add certain Build Phases and Scheme changes to the Test Target. We recommend using our earlgrey gem to install any dependencies if you're using a CocoaPods version from `1.0.0` onwards. Run `gem install earlgrey`. You need to require this gem in a post_install hook using the project's name, the test target's name, and the name of the xcscheme file as is shown in the below example for `1.0.0`

You can move this passage to above the 0.39.0 example -

> Since the `earlgrey gem` is supported only beyond CocoaPods 1.0.0 or if you don't want a gem dependency, replace require 'earlgrey' with load 'configure_earlgrey.rb' and manually copy configure_earlgrey.rb from the gem into the same directory as the Podfile.

**Comment by gurgeous** [comment]
How about this?

File: docs/install-and-run.md [hunk 1]
```diff
@@ -64,6 +61,8 @@ For EarlGrey, we highly recommend [CocoaPods](https://cocoapods.org/) as the bes
   end
   ```
 
+    The earlgrey gem is supported with CocoaPods 1.0.0 and beyond. For earlier versions (or if you don't want a gem dependency) replace `require 'earlgrey'` with `load 'configure_earlgrey.rb'` and manually copy `configure_earlgrey.rb` from the gem into the same directory as the `Podfile`.
+
```
Comment (khandpur) [inline docs/install-and-run.md]: This looks good but I'd also replace the actual code (in line # 79) with `load 'configure_earlgrey.rb'` for v0.39 and below. thanks!

**Comment by gurgeous** [comment]
better?

**Comment by bootstraponline** [comment]
Looks good. :+1:

**Comment by khandpur** [comment]
Merging this. Thanks Adam!

**Pull request merged**